docs/src/pages/ThemingSpecification.mdx

```diff
@@ -0,0 +1,297 @@
+export const meta = {
+  title: 'Theming specification',
+}
+
+## Goals and requirements
+
+The theming mechanism as part of Fluent UI should be flexible and provide support for CSSinJS theme, and all existing customization scenarios. It must also allow users to opt-out if the theming approach does not meet their needs. This means that we need to have support for both CSS stylesheets and CSSinJS.
+
+##### Support for CSS stylesheets
+
+Тhe users should be able to specify mappings from properties to classnames
```

Comment:
I love this goal. Thanks for adding.

Just anecdotally; VSTS ended up with a forked version of Fabric; one of the reasons was because they wanted to use pure css for perf. We have ODSP saying they want 60ms initial page render times. Every single run time overhead we can remove is great.